app/services/integrations/ltsa_parcel_map_bc.rb

```diff
@@ -10,8 +15,37 @@ def get_details_by_pid(
     pid:,
     fields: "PID,PARCEL_STATUS,PARCEL_NAME,PARCEL_CLASS,OWNER_TYPE,MUNICIPALITY,REGIONAL_DISTRICT,WHEN_UPDATED,FEATURE_AREA_SQM"
   )
-    @client.get(
-      "query?f=json&returnIdsOnly=false&returnCountOnly=false&where=PID='#{pid}'&returnGeometry=true&spatialRel=esriSpatialRelIntersects&outFields=#{fields}",
-    )
+    query = "returnIdsOnly=false&returnCountOnly=false"
```

Comment:
Do you think it might be worth it to refactor this to use my wrappers/ folder and it's base superclass?
This integrations folder seems to be a repeated concept and is not using the built in faraday conventions for providing query params

Please see wrappers/base.rb and its get method, it would be a good opportunity to cut down on clutter and dry things up